Solutions/Delinea Secret Server/Workbooks/DelineaWorkbook.json

```diff
@@ -315,8 +315,8 @@
     }
   ],
   "fallbackResourceIds": [
-    ""
+    "/subscriptions/80e05ab0-5906-45a8-98ff-c9520c4b418b/resourcegroups/thycotic/providers/microsoft.operationalinsights/workspaces/thycotic02"
```

Comment:
@gaurava-delinea, @tylerezimmerman, workbook validations are failing as the workbook contains info of a resource. A workbook template must not contain any references to resources. Please check and update, thanks.

Reply:
> @gaurava-delinea, @tylerezimmerman, workbook validations are failing as the workbook contains info of a resource. A workbook template must not contain any references to resources. Please check and update, thanks.

Hey @gaurava-delinea , @tylerezimmerman , can you please fix this, all validations should pass for the PR to be merged.